ports/stm/boards/STM32H743ZITx_FLASH.ld

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
Eventually (not now), take a look at how I came up with a common templated linker script for `atmel-samd` and `nrf`. There's a mechanism for extracting some `#define` values from the `.h` files and then filling in needed values in the `.ld` file.

Reply:
I'm actually working on that already today - whoops

Reply:
Do the STM32HAL files have these values? I took a different approach for iMX RT because it's SDK didn't have the info (and flash size is board dependent.)